forch/port_state_manager.py

```diff
@@ -188,6 +192,28 @@ def handle_static_device_behavior(self, mac, device_behavior):
             self._auto_sequester[mac_lower] = auto_sequester
             if device_behavior.segment:
                 self.handle_device_behavior(mac_lower, device_behavior, static=True)
+            sequester_scheduled = self._scheduled_sequester_timer.get(mac_lower)
+            if device_behavior.scheduled_sequestering_timestamp and not sequester_scheduled:
+                try:
+                    parsed = dateutil.parser.parse(device_behavior.scheduled_sequestering_timestamp)
+                except dateutil.parser.ParserError:
+                    self._logger.error("Failed to parse scheduled sequestering timestamp: %s.",
+                                       device_behavior.scheduled_sequestering_timestamp)
+                    return
+                local_tz = dateutil.tz.tzlocal()
+                if not parsed.tzinfo:
```

Comment:
NB: There is a race (?) condition here where the timeout won't happen if the system happens to be restarting when the sequestered is scheduled.  but -- I don't care about that so I don't think it's worth addressing -- just throwing the comment in for completeness. Maybe add a TODO or something acknowledging this.